packages/code-du-travail-data/indexing/fetchCdtnAdminDocuments.js

```diff
@@ -0,0 +1,82 @@
+import { getRouteBySource, SOURCES } from "@socialgouv/cdtn-sources";
+import allThemes from "@socialgouv/datafiller-data/data/themes.json";
+import fetch from "node-fetch";
+
+import { createThemer } from "./breadcrumbs";
+
+const getBreadcrumbs = createThemer(allThemes);
+const fixBreadcrumbs = (source) => {
+  if (source === SOURCES.EDITORIAL_CONTENT) {
+    return [
+      {
+        label: "Dossier Coronavirus-Covid 19",
+        slug: `/${getRouteBySource(
+          SOURCES.THEMATIC_FILES
+        )}/ministere-du-travail-notre-dossier-sur-le-coronavirus`,
+      },
+    ];
+  }
+  return [];
+};
+const CDTN_ADMIN_ENDPOINT =
+  process.env.CDTN_ADMIN_ENDPOINT || "http://localhost:3000/api/graphql";
+
+const gqlRequestBySource = (source) =>
+  JSON.stringify({
+    query: `{
+  documents(where: {source: {_eq: "${source}"}}){
+    id:initial_id
```

Comment:
could we have standard naming convention here ? or there is good reason to have different names in PGSQL and ES ?

Reply:
I generally use snake_case for sql table field and camelCase pour Js